### PR TITLE
fix issue in sendMediaGroup

### PR DESCRIPTION
**Pull request by alr2413**
In the "sendMediaGroup" method the "inputFileField" key has been incorrectly set as "video_note" instead of "media". Also, since the "media" field is JSON serialized string and not an input field, as mentioned in the Telegram API Docs, I updated the "validateInputFileField" in "Traits/Http.php" class to support this case.
Thanks.

### CODE REVIEW

File: src/Traits/Http.php [hunk 1]
```diff
@@ -342,7 +342,7 @@ protected function validateInputFileField(array $params, $inputFileField)
         }
 
         //All file-paths, urls, or file resources should be provided by using the InputFile object
-        if (! $params[$inputFileField] instanceof InputFile) {
+        if ((!$params[$inputFileField] instanceof InputFile) && (!is_json($params[$inputFileField]))) {
```
Comment (irazasyed) [inline src/Traits/Http.php]: PHP doesn't have `is_json` method.

Comment (alr2413) [reply src/Traits/Http.php]: Yes, I checked my code again and found that it is a function from another library with the following code.

```
function is_json($string)
{
	json_decode($string);
	return (json_last_error() == JSON_ERROR_NONE);
}
```

**Comment by alr2413** [comment]
Please Reply my question:

Yes, I checked my code and found that it is a function from another library with the following code. Where should I place this function?

```
function is_json($string)
{
	json_decode($string);
	return (json_last_error() == JSON_ERROR_NONE);
}
```

_Originally posted by @alr2413 in https://github.com/irazasyed/telegram-bot-sdk/pull/691_

File: src/Traits/Http.php (hunk 1 above)
Comment (irazasyed) [reply src/Traits/Http.php]: Please add it to the [Validator](https://github.com/irazasyed/telegram-bot-sdk/blob/develop/src/Traits/Validator.php) trait.

Comment (alr2413) [reply src/Traits/Http.php]: Thanks.
I've added this change.
Please check it again.

**Comment by irazasyed** [comment]
Thanks!

**Pull request merged**